### PR TITLE
use telefunc for api methods

### PR DESCRIPTION
**Pull request by nathanstitt**
Removes the need for express and lambda duplication and is more convenient

### CODE REVIEW

File: server/service.together.ts [hunk 1]
```diff
@@ -60,6 +60,7 @@ export async function requestInference(ctx: InferenceContext) {
                 controller.abort()
                 throw new CompletedError()
             }
+            console.log(chunk.data)
```
Comment (chrisbendel) [inline server/service.together.ts:63]: keeping intentionally?

File: server/lambda_api.ts [hunk 2]
```diff
@@ -1,8 +1,12 @@
...
-import { userEligibility } from '#server/user'
+import { getUserFromCookie } from './authentication'
+import { setUsageTime } from '#server/user'
+
+
+// import { RequestContext } from './request-context.js'
+// import { userEligibility } from '#server/user'
+import { addTelefuncHandlerToApp } from './middleware'
 
 // @Developer reading this
```
Comment (chrisbendel) [inline server/lambda_api.ts:11]: this is moot now with telefunc right?

File: package.json [hunk 3]
```diff
@@ -98,6 +100,7 @@
   "devDependencies": {
     "@playwright/test": "^1.38.1",
     "@testing-library/react-hooks": "^8.0.1",
+    "buffer": "^5.5.0||^6.0.0",
```
Comment (chrisbendel) [inline package.json:103]: i might've missed where this was used in the PR but just curious what buffer is used for

**Review by chrisbendel — APPROVED** [review]

File: server/service.together.ts (hunk 1 above)
Comment (nathanstitt) [reply server/service.together.ts:63]: yeah, I removed in a the following pr once I figured the error out

File: server/lambda_api.ts (hunk 2 above)
Comment (nathanstitt) [reply server/lambda_api.ts:11]: hah yeah, I'll remove

File: package.json (hunk 3 above)
Comment (nathanstitt) [reply package.json:103]: hm, I'm actually not sure how that got here. I'll remove and see if anything breaks

**Pull request merged**